src/lattice.py

```diff
@@ -50,19 +59,28 @@ async def publish_entity(self, entity) -> Optional[Dict[str, Any]]:
         Raises:
             None
         """
+        session: Optional[aiohttp.ClientSession] = None
```

Comment:
The auto formatter obscures some of the changes here, but the changes are effectively these 2 lines, and the changes to line 76-77.